### ISSUE TITLE
Include popular forks.

### ISSUE THREAD

**Issue by phrohdoh** [resume/resume.github.com#189]
The resume should not only count source repos but popular forks also.
This makes the page just a bit more accurate (or much more accurate, in my case).

**Comment by u2** [comment]
Agreed with @nicknovitski , and this will also run out of the RateLimit.

**Comment by HairyFotr** [comment]
How about a variant of this feature that doesn't require deep checking: show the forks that have, for example, more than 15 stars by themselves.

**Comment by HairyFotr** [comment]
I'd like to add this for anyone else who finds this issue: For major forks, that either become the leading repo of a project, or for forks that change the purpose of the project, it's possible to contact GitHub support and request an "unforking". All history and data is preserved.
